keep/api/arq_worker.py

```diff
@@ -84,7 +83,7 @@ def get_arq_worker(queue_name: str) -> Worker:
         "ARQ_KEEP_RESULT", cast=int, default=3600
     )  # duration to keep job results for
     expires = config(
-        "ARQ_EXPIRES", cast=int, default=3600
+        "ARQ_EXPIRES", cast=int, default=3600*1000 if KEEP_ARQ_TASK_POOL == KEEP_ARQ_TASK_POOL_AI else 3600
```

Comment:
when someone runs Keep with `ARQ_EXPIRES` it overrides the default, what will happen to your AI? if your AI needs more time? you need to warn somehow that expiration is too low. Also, its not clear to someone who will read the code why you did that.

Reply:
It's a copy of the condition introduced by @Matvey-Kuk [here](https://github.com/keephq/keep/blob/9f3895985e78373dfda54d9a8395e4b4566e538c/keep/api/arq_worker.py#L122).

I'm not sure I get the point, do you propose to have extra logs here, introduce a better config for ai-related operations, add documentation?

Reply:
1. dont you see the difference? what you sent from @Matvey-Kuk is a hardcoded value. here its a value from environment value. you changed only the default value when environment value is not supplied. can you tell which user deploys Keep with `ARQ_EXPIRES`?
2. regarding logs and better explainability - that's a goes without saying

Reply:
But .ee is not supposed to be used by any external users or am I missing something?

Re extended expiration: I think in this case we need to go back to discussion about possible caveats of an extended expiration time with @talboren

Reply:
@shahargl I added extra logs ofc

Reply:
I dig a bit into `expires` which is getting assigned to `expires_extra_ms` (check here https://arq-docs.helpmanual.io/). That this value is responsible for task discharging in case old tasks are kept in the queue for too long. It's a time = time_task_scheduled_for - time_task_started. Since it's `time_task_started`, it shouldn't be different for long-lasting or short-lasting tasks.

I propose to keep this value the same for AI or non-AI tasks :)

Reply:
Problem resolved with the introduction of ARQ_EXPIRES_AI variable